docs/java.md

```diff
@@ -0,0 +1,373 @@
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.PlainTextAuthProvider;
+import com.datastax.driver.core.policies.DCAwareRoundRobinPolicy;
+
+import java.util.Date;
+import java.util.Scanner;
+
+class Main {  
+  
+    public static void main(String[] args) { 
+        Cluster cluster = Cluster.builder()  
+            .addContactPoints("your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud")  
+            .withLoadBalancingPolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("AWS_US_EAST_1").build()) // your local data center  
+            .withAuthProvider(new PlainTextAuthProvider("scylla", "*******"))  
+            .build();  
+            
+            Session session = cluster.connect();  
+
+            PreparedStatement statement = session.prepare(
+                "UPDATE songs set title = ? where id = ? AND updated_at = ?"
+            );
+
+            BoundStatement bound = statement.bind()
+                    .setLong(0, 1)
+                    .setString(1, "Rock and Roll")                    
+                    .setTimestamp(2, new Date());
+
+            session.execute(bound);
+     }
+}
+```
+
+After the data gets inserted, query all columns and filter by the ID:
+```
+scylla@cqlsh:media_player> select * from songs where id = 1;
+
+id | updated_at                      | album       | artist | created_at                      | title
+---+---------------------------------+-------------+--------+---------------------------------+----------------------------
+ 1 | 2023-03-02 22:00:00.000000+0000 | Smithereens |   Joji | 2023-03-02 22:00:00.000000+0000 |              Glimpse of Us
+ 1 | 2023-03-02 23:10:00.000000+0000 |        null |   null |                            null | Glimpse of US - Inutilismo
+```
+
+It only updated the field `title` and `updated_at` (the Clustering Key) and since we didn't input the rest of the data, it will not be replicated as expected.
+
+
+### 3.5 Deleting Data
+
+Last things last! Let's understand what we can DELETE with this statement. There's the normal `DELETE` statement that focus on `ROWS` and other one that delete data only from `COLUMNS` and the syntax is very similar.
+
+```sql 
+// Deletes a single row
+DELETE FROM songs WHERE id = d754f8d5-e037-4898-af75-44587b9cc424;
+
+// Deletes a whole column
+DELETE artist FROM songs WHERE id = d754f8d5-e037-4898-af75-44587b9cc424;
+```
+
+If you want to erase a specific column, you also should pass as parameter the `Clustering Key` and be very specific in which register you want to delete something. 
+On the other hand, the "normal delete" just need the `Partition Key` to handle it. Just remember: if you use the statement "DELETE FROM <table>" it will delete ALL the rows that you stored with that ID. 
```

Comment:
Not sure why its normal.
Maybe parition delete?

Reply:
I suggested "regular" instead of "normal" above. Anyway, we should avoid using "normal" in the docs.